packages/js/components/src/experimental-tree-control/hooks/use-tree-item.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { TreeItemProps } from '../types';
+
+export function useTreeItem( { item, level, ...props }: TreeItemProps ) {
+	const nextLevel = level + 1;
+	const nextHeadingPaddingLeft = ( level - 1 ) * 28 + 12;
```

Comment:
Could we apply these styles via SCSS?  This might give us more flexibility for consumers in the future.